srcpkgs/vala-panel/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'vala-panel'
+pkgname=vala-panel
+version=0.4.87
+revision=1
+build_style=cmake
+hostmakedepends="pkgconf"
+makedepends="vala-devel cmake-vala gtk+3-devel libglib-devel"
+short_desc="GTK+3 desktop panel inspired by LXPanel"
+maintainer="John Eira <john.eira@web.de>"
+license="LGPL-3.0-or-later"
+homepage="https://gitlab.com/vala-panel-project/vala-panel"
+distfiles="https://gitlab.com/vala-panel-project/vala-panel/uploads/8b42536f91a1d6a185350ce2e3b7e91f/vala-panel-${version}.tar.xz"
+checksum=625c12672553e1807f98f58c43c631c081938652756daddf0f6e37cffe392d08
+
+vala-panel-devel_package() {
+	short_desc+=" - development files"
+	depends="vala-panel"
```

Comment:
please restrict the version like it is done in other -devel packages

Reply:
ok